hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -129,8 +129,6 @@ public static class HiveSyncConfigParams {
     public String bucketSpec;
     @Parameter(names = {"--sync-comment"}, description = "synchronize table comments to hive")
     public Boolean syncComment;
-    @Parameter(names = {"--with-operation-field"}, description = "Whether to include the '_hoodie_operation' field in the metadata fields")
-    public Boolean withOperationField; // TODO remove this as it's not used
```

Comment:
Is this change related to this PR?

Reply:
Since this variable isn't used, I thought it would be a good idea to remove it.

@fengjian428 for visibility

Reply:
OK, I just have some concerns:

- Let's make one PR focus on one thing. Keep it simple. WDYT?
- Anytime we remove a public config option, we must consider compatibility seriously. If there is a compatibility issue, a better choice is to make it with `@Deprecated` and remove it after two major releases.

Reply:
Thank you for the explanation. 
Agree with what you say! Will revert the changes for this config.